app/javascript/app/components/header/header-styles.scss

```diff
@@ -119,6 +119,8 @@ $flag-height: 40px;
     top: 20px;
   }
 
+  overflow: scroll;
```

Comment:
This `overflow` property is forcing the display of some scrollbars on `x` & `y` axis of `.tabsContainer`

![image](https://user-images.githubusercontent.com/6906348/45033804-f8579c00-b055-11e8-8427-95c3404fdac2.png)

Reply:
I think it's worth also adding `white-space: nowrap;` for the tab names, there are some responsive issues with those

Reply:
For these, I think we need a new design in responsive. Some tabs have three elements with several lines which we wouldn't be able to show in a nice way

Reply:
I will use overflow-x and cancel it on tablet-landscape but we have to keep in mind that this should'nt be the last solution as the active tab sometimes gets out of the screen and the scrolling is not obvious for the user

Reply:
ok, @Bluesmile82, maybe add a TODO in the code to have that in mind in the future